recipes/libsvm/all/conanfile.py

```diff
@@ -37,29 +39,31 @@ def validate(self):
             self.options.shared
         ):
             raise ConanInvalidConfiguration(
-                "{} can not be built as shared library + runtime {}.".format(
-                    self.name,
-                    self.settings.compiler.runtime
-                )
+                f"{self.name} can not be built as shared library + runtime {self.settings.compiler.runtime}."
             )
 
+    def layout(self):
+        cmake_layout(self, src_folder="source_subfolder")
```

Comment:
For consistency, best to name the source folder `src`.

```suggestion
        cmake_layout(self, src_folder="src")
```

Reply:
Consistency with what? Other conan recipes?
I kept the `source_subfodler` name since it is hardcoded on the `CMakeLists.txt`. I can update this there as well if you feel it would be appropriate.

Reply:
I've been seeing the following idiom where the source directory is passed to CMake as a variable from the conanfile. The `quickjs` recipe was just updated to do this: https://github.com/conan-io/conan-center-index/blob/dbf9f46286fb44ea7cd4a54ebb5347a63eb3e3f2/recipes/quickjs/all/conanfile.py#L67

I think it's probably a worthwhile enhancement to the recipe since it doesn't rely on a specific name for the source directory to be hard-coded in the CMakeLists.txt file.

Reply:
Well, I don't love it but I'm all in for consistency. Done on the last commit :)